resources/public/api/conf/1.0/listCharges.yaml

```diff
@@ -0,0 +1,96 @@
+get:
+  summary: List Self Assessment Charges [test only]
+  description: |
+    This endpoint allows a developer to retrieve all details of a user's business income sources. The unique account reference (National Insurance number) must be provided.
```

Comment:
I don't think this is the right description - it doesn't match the description in listCharges.raml: 
<img width="684" alt="image" src="https://user-images.githubusercontent.com/106095575/215825128-bb17db8a-df27-4730-afbf-3c8f8c21e5ff.png">